tests/acceptance/bootstrap/SpacesTUSContext.php

```diff
@@ -90,6 +90,25 @@ public function userUploadsAFileViaTusInsideOfTheSpaceUsingTheWebdavApi(
 		$this->featureContext->setLastUploadDeleteTime(\time());
 	}
 
+	/**
+	 * @When the public uploads file :source to :destination via TUS inside last link shared folder with password :password using the WebDAV API
+	 *
+	 * @param string $source
+	 * @param string $destination
+	 * @param string $password
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function publicUploadsAFileViaTusInsideOfTheSpaceUsingTheWebdavApi(
```

Comment:
```suggestion
	public function thePublicUploadsFileViaTusInsideLastSharedFolderWithPasswordUsingTheWebdavApi(
```
Make sure the function name matches step definition.